source/_patterns/01-molecules/teaser/teaser.twig

```diff
@@ -0,0 +1,18 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser']) }) -%}
+
+<div {%- include('atoms-attributes') -%}>
+
+  {%- with title|default({}) only -%}
+    {%- block title -%}
+
+      {% if text is defined %}
+
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser__title']) }) -%}
+        {%- include 'atoms-heading' with {level: level|default(4)} -%}
```

Comment:
Currently. `heading` supports level and text only. We will want to adjust this so `heading` can support links and some inline styles.